packages/base-styles/_variables.scss

```diff
@@ -71,6 +71,7 @@ $shadow-modal: 0 3px 30px rgba($black, 0.2);
  */
 
 $sidebar-width: 280px;
+$navigation-editor-toolbar-width: $sidebar-width;
```

Comment:
We could just use `$sidebar-width` directly, or else give the new variable a more generic name such as `$toolbar-width`. I notice the `280px` value is hardcoded for a few of the post editor toolbars so perhaps a more generic variable would be useful.